neo/Consensus/ConsensusService.cs

```diff
@@ -82,8 +84,30 @@ private void CheckExpectedView(byte view_number)
 
         private void CheckSignatures()
         {
+            if (!context.State.HasFlag(ConsensusState.CommitSent) &&
```

Comment:
@shargon,
I believe that here we should check the partial signatures instead if the complete and, then, at lines 353 (https://github.com/neo-project/neo/blob/064ebc92f6617f6d25234214ef80ef386e34ff2b/neo/Consensus/ConsensusService.cs#L353) and 52 (https://github.com/neo-project/neo/blob/064ebc92f6617f6d25234214ef80ef386e34ff2b/neo/Consensus/ConsensusService.cs#L52) they should not be send.